### ISSUE TITLE
GameGuru MAX - Problem with Terrain - Random hole.

### ISSUE THREAD

**Issue by jasailor** [TheGameCreators/GameGuruRepo#4095]
This is the second time that one of my MAX (experimental) levels has been corrupted with this hole in the terrain.  I tried the various terrain tools to fix it but they don't correct the issue.  The last time I had to start the level over.    Can someone provide a way to fix it and let me know if I am doing something wrong that could be causing it?
+

![Terrain Issue](https://user-images.githubusercontent.com/41845014/213298102-fdf45f41-2e1c-43ef-9db6-80d29f95f3e3.PNG)

**Comment by davetgc** [comment]
Is this happening when placing objects from the building editor?

**Comment by jasailor** [comment]
Nope.   I was only doing some work with the terrain editor.   Last night I went down the hole within the editor and found the hanger from the  shanty town collection (actually not an object in the level) and a number of palm trees .   While in the hole, I deleted the assets,  then used the “raise tool” to create a peek from within the hole to above the surface and finally used the smoothing tool to  raise the balance of the hole to the surface level.   Seem to solve the problem this time.   The first time this occurred a while ago I wasn’t able to manipulate it and had to abandoned the level. Kind a weird.  I live in FL where there are sink holes, but I didn’t expect to experience them in the virtual world too!  😂  Let me know if you need anything to help find the cause.Regards,JASailorOn Jan 19, 2023, at 12:58 AM, davetgc ***@***.***> wrote:﻿
Is this happening when placing objects from the building editor?

**Comment by davetgc** [comment]
I'm marking this as a medium bug until we can get some more information on the cause. If you find any more information, please update this thread.

**Comment by ZakJudges** [comment]
Hi @jasailor, so I can investigate this, please could you email me the level from the mapbank folder that shows the hole in the terrain? My email address is zak@thegamecreators.com. Thanks

**Comment by Hamada880** [comment]
I can confirm it happen a few weeks ago, I happen to add building was made through GGMax building tool and I removed it, hit save and it created the hole. I went back to try the same thing in a new scene,but I wasn't able to create the same bug again.

**Comment by AndrewMarais** [comment]
I was able to make it on the experimental build
[https://youtu.be/6tTx2qlMlLY](url)

**Comment by ZakJudges** [comment]
@AndrewMarais Thank you for the video. I have added a fix which should hopefully be in the next experimental build. I will post here when the new experimental build is released :)

**Comment by AndrewMarais** [comment]
@ZakJudges I tried that level again just now and still had that hole. So then it got interesting. 
When I removed the model of the big building, the hole got smaller (after reloading). So I removed the other smaller building model (also to windows recycle bin)

**The black hole in the level disapeared!** (restoring the original height also) So it appears to be affecting when the level is initally loaded. 
HTH
